editor/editor_node.cpp

```diff
@@ -4120,10 +4131,118 @@ void EditorNode::update_ownership_table_for_addition_node_ancestors(Node *p_curr
 	}
 }
 
-void EditorNode::update_diff_data_for_node(
-		Node *p_edited_scene,
+void EditorNode::update_node_from_node_modification_entry(Node *p_node, ModificationNodeEntry &p_node_modification) {
+	if (p_node) {
+		// First, attempt to restore the script property since it may affect the get_property_list method.
+		Variant *script_table_entry = p_node_modification.property_table.getptr(CoreStringNames::get_singleton()->_to_string);
```

Comment:
```suggestion
		Variant *script_table_entry = p_node_modification.property_table.getptr(CoreStringName(_to_string));
```
Though I assume you meant to fetch the `script` property:
```suggestion
		Variant *script_table_entry = p_node_modification.property_table.getptr(CoreStringName(script));
```

Reply:
Okay, I fixed this to use the correct StringName now, the change to use format `CoreStringName(script));` doesn't seem to work. Could you elaborate?

Reply:
Your branch is 800 commits behind `master`, so you're missing a number of important changes there, notably #91909.

So you should rebase, and then you can use what @AThousandShips suggested.